Module1/todo_app/app.py

```diff
@@ -2,13 +2,26 @@
 
 from todo_app.flask_config import Config
 from todo_app.data.session_items import get_items, add_item, save_item
+from todo_app.data.trello_items import *
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
+API_KEY = os.getenv('API_KEY')
+API_TOKEN = os.getenv('API_TOKEN')
+BOARD_NAME = os.getenv('BOARD_NAME')
 
-@app.route('/',methods = ['POST', 'GET'])
-def index():
+""" BOARD_LIST = {
+    'To_Do': '64ef5905b30edf49e85fa346', 
+    'Doing': '64ef5905b30edf49e85fa347',
+    'Done': '64ef5905b30edf49e85fa348'
+}   """
```

Comment:
I think lines 12-18 are all no longer needed, and it's a good habit to get into cleaning these out rather than just commenting them - we can always recover them with Git if needed, but they're adding cruft and confusion to the codebase currently